sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -432,6 +451,70 @@ async Task GetDetails(MessageId msg_id, MonoCommands cmd, CancellationToken toke
 			}
 
 		}
+		
+		
+		public async Task<JObject> GetVariableValue (MessageId msg_id, int scope_id, string expression, CancellationToken token)
+		{
+			JObject thisValue = null;
+			var context = GetContext (msg_id);
+			if (context.CallStack == null)
+				return null;
+			var scope = context.CallStack.FirstOrDefault (s => s.Id == scope_id);
+			var vars = scope.Method.GetLiveVarsAt (scope.Location.CliLocation.Offset);
+			//get_this
+			int i = 0;
+			int [] var_ids = { };
+			var res = await SendMonoCommand (msg_id, MonoCommands.GetScopeVariables (scope.Id, var_ids), token);
+			var values = res.Value? ["result"]? ["value"]?.Values<JObject> ().ToArray ();
+			for (; i < values.Length; i++) {
```

Comment:
Maybe, `thisValue = values.Where (v => v ["name"].Value<string> () == "this").FirstOrDefault ()`? unless this is faster.